tests/test_bindings.py

```diff
@@ -576,3 +576,121 @@ def test_scalarmult_ed25519_base():
     p2 = c.crypto_scalarmult_ed25519(sclr, BASEPOINT)
 
     assert p2 == p
+
+
+def test_scalarmult_ed25519_noclamp():
+    one = 32 * b'\x01'
```

Comment:
I don't really like calling this big integer `one`, especially since this differs from the following `one`s.

Reply:
Yes agreed... the intent was to use a value which is known to differ once clamped.
Would it work if I replace `one` and `one_clamped` with resp. `scalar` and `scalar_clamped` ?

```
def test_scalarmult_ed25519_noclamp():
    # An arbitrary scalar which is known to differ once clamped
    scalar = 32 * b'\x01'
[...]
```

Reply:
Anything replacing `one` goes! OK for `scalar`